0-ad-history-mod-package/simulation/templates/units/mace/champion_infantry_crossbowman.xml

```diff
@@ -5,6 +5,12 @@
     <Lang>greek</Lang>
     <SpecificName>Gastraphetophoros</SpecificName>
     <Icon>units/mace/champion_infantry_crossbowman.png</Icon>
+    <History>
+        The gastraphetes was a type of Greek crossbow, developed during the 5th or 4th century BC. Its name, deriving from its unique design, directly translates to “belly shooter” and, in a broader context, to “belly bow” which is the term used today.
+        The bow itself and the claw mechanism grabbing the bowstring were attached on opposite ends of two different beams of similar length. One of those beams (the one with the claw) rested inside a groove directly on the other beam, allowing both to slide back and forth along the other. With this particular arrangement, the bow and the claw mechanism could easily be brought together by pulling the top beam and/or pushing the lower beam. However, instead of then pulling back the bowstring like most crossbows do, the gastraphetes actually pushed the bow away from its string to create the tension: The upper beam, now protruding from the lower beam, was placed at a 45-degree angle forward on the ground. A big concave piece of wood attached to the lower beam's rear was placed to the shooter's belly. By leaning forward and pushing with his hands, pressing the crossbow against the ground, the shooter moved the beams together, therefore the bow away from its string, bending it, and therefore drawing the crossbow.
+        The big strength of this design lied in its simplicity of use. Even people who had never held a gastraphete in their hands before could, with as little as only a couple of days or weeks of training, learn the basic movement cycle and fight with it. The pushing motion against the ground utilizing bodyweight, also enabled even quite untrained and physically weak soldiers to draw a strong gastraphetes. For the same reason, it had a very high draw weight (probably over 100 lbs), which subsequently led to great momentum on the arrow, a incredible range and penetrating power.
+        One major drawback of this simple and efficient loading mechanism was its speed. The gastraphetes could only fire an estimated one or two arrows per minute, which was far slower than many other ranged weapons at the time. It was also, because it was fired while held similarly to the belly, harder to aim at a distant target than other crossbows. For this reason, as well as due to its large weight and mechanical complexity, the gastraphetes was (sadly) not very widely used.
```

Comment:
```suggestion
        One major drawback of this simple and efficient loading mechanism was its speed. The gastraphetes could only fire an estimated one or two arrows per minute, which was far slower than many other ranged weapons at the time. It was also, because it was fired while held similarly to the belly, harder to aim at a distant target than other crossbows. For this reason, as well as due to its large weight and mechanical complexity, the gastraphetes was (sadly) not widely used.
```